README.md

```diff
@@ -24,7 +24,7 @@
     - [`<Endpoint/>`](#endpoint)
     - [`<EndpointsTable/>`](#endpointstable)
     - [`<ExampleResponse/>`](#example-json-response)
-    - [`<NavTable/>`](#navtable)
+    - [`<MethodTable/>`](#MethodTable)
```

Comment:
`NavTable` was just a thin wrapper over `MethodTable`, so I removed the extra component.